AUTHORING_GUIDE.md

```diff
@@ -46,7 +46,7 @@ owner to review your code for product-specific needs.
 In **library repositories** with pre-existing samples GitHub will automatically assign a reviewer
 from python-samples-owners. If no reviewer is automatically assigned, contact [@googleapis/python-samples-owners](https://github.com/orgs/googleapis/teams/python-samples-owners).
 
-Feel free to reach out to your assigned reviewer if it's been a few days and you haven't gotten a response!
+Please reach out to your assigned reviewer if it's been more than 2 days and you haven't gotten a response!
```

Comment:
Switched to "2 days" to match [this doc](https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/reviews.md?cl=head#samples-reviewers).